spec/requests/api/v1/items_requests_spec.rb

```diff
@@ -265,4 +265,51 @@
     end
   end
 
+  describe "Can create items sad paths" do 
+    it "can't create new items with no price" do 
+      attributes = {
+        name: "chocolate bar",
+        description: "sweet and delicious",
+        merchant_id: 1
+      }
+    
+      post "/api/v1/items", params:{item: attributes}
```

Comment:
At first I was a little confused about this test but upon further analyzation I can see that the POST request to create an item fails without a price attribute.
The GET request still succeeds afterward, indicating normal functionality. Looks good!